### PR TITLE
Add infomration to use Source Side Validation script

### PR DESCRIPTION
**Pull request by Batre-MSFT**
Added references to use the source side validation script

Line # 31 -> SSV script's purpose 
Line # 102 -> location to download SSV
Line # 120 -> Add an additional step here suggesting to run Source Side Validation script at mailbox server on Exchange Server 2010. Please use the examples as documented here: https://techcommunity.microsoft.com/t5/Exchange-Team-Blog/Making-your-public-folder-migrations-faster-and-more-reliable/ba-p/917622

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/batch-migration-of-legacy-public-folders.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ We recommend that you don't use Outlook's PST export feature to migrate public f
 
 You'll perform the migration using the **\*-MigrationBatch** cmdlets, in addition to the following PowerShell scripts:
 
+- ` ssv.ps1`:  Source Side Validation script scans the public folders at source and reports issues found along with action to fix the issues. You'll run this script on the legacy Exchange server On-Premises.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/batch-migration-of-legacy-public-folders.md]:
```suggestion
- ` ssv.ps1`:  Source Side Validation script scans the public folders at source and reports issues found along with actions required to fix the issues. You'll run this script on the legacy Exchange server on-premises.
```

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/batch-migration-of-legacy-public-folders.md [hunk 2]
```diff
@@ -97,7 +99,9 @@ You can't migrate public folders directly from Exchange 2003 or Exchange 2007. I
 
     - `SyncMailPublicFolders.strings.psd1`
 
-4. Save the scripts to the same location you did for step 2. For example, C:\PFScripts.
+4. Download the source side validation script from https://www.microsoft.com/en-us/download/confirmation.aspx?id=100414
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/batch-migration-of-legacy-public-folders.md]:
```suggestion
4. Download the source side validation script from the [Microsoft Download Center](https://www.microsoft.com/download/confirmation.aspx?id=100414). 
```

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/batch-migration-of-legacy-public-folders.md [hunk 3]
```diff
@@ -97,7 +99,9 @@ You can't migrate public folders directly from Exchange 2003 or Exchange 2007. I
...
+
+5. Save the scripts to the same location you did for step 2. For example, C:\PFScripts.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/batch-migration-of-legacy-public-folders.md]:
```suggestion
5. Save the scripts to the same location as you did in step 2, for example, C:\PFScripts.
```

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/batch-migration-of-legacy-public-folders.md [hunk 4]
```diff
@@ -113,6 +117,8 @@ Perform the following prerequisite steps before you begin the migration.
 
 ### Prerequisite steps on the legacy Exchange server
 
+Add an additional step here suggesting to run Source Side Validation script at mailbox server on Exchange Server 2010. Please use the examples as documented here: https://techcommunity.microsoft.com/t5/Exchange-Team-Blog/Making-your-public-folder-migrations-faster-and-more-reliable/ba-p/917622
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/batch-migration-of-legacy-public-folders.md]:
```suggestion
We recommend running Source Side Validation script from a mailbox server on Exchange Server 2010. Please use the examples as documented [here](https://techcommunity.microsoft.com/t5/Exchange-Team-Blog/Making-your-public-folder-migrations-faster-and-more-reliable/ba-p/917622).
```

**Pull request merged**